docs/source/configuration.rst

```diff
@@ -16,9 +16,9 @@ Configuring Calico
 ==================
 
 This page describes how to configure Calico. We first describe the
-configuration of the core Calico component - Felix -
+configuration of the core Calico component -- Felix --
 because this is needed, and configured similarly, regardless of the
-surrounding environment (OpenStack, Docker, or whatever). Then,
+surrounding environment (OpenStack, Docker, or whatever.) Then,
```

Comment:
I thought the rule was that you should be able to take the part in parentheses away without breaking the sentence - i.e. in this case, taking away the parentheses section will take out the full stop, which is wrong.